lib/item.js

```diff
@@ -21,7 +21,7 @@ function Item(obj, collection, app) {
   utils.defineProp(this, 'collection', collection);
   utils.defineProp(this, 'app', app);
   this.history = [];
-  return obj;
+  this.visit('set', obj);
```

Comment:
@jonschlinkert what do you think of this? I think it makes more sense than returning the passed in object without doing anything to it. This will allow anyone to use `Item` directly without having to do some of the special things that `View` does (like `view.__proto__ = this`)